### ISSUE TITLE
transcribed video not showing video in transcript editor and programme script

### ISSUE THREAD

**Issue by sarahrainbow** [bbc/digital-paper-edit-firebase#294]
low res footage uploaded by Tony Ellam 'TURBINES_FLEURY-ProRes Proxy half size.mov'
will only playback audio in both transcript editor and programme script view.

It seems he got low res version by exporting from Jupiter.

**Comment by allishultes** [comment]
It isn't possible to preview this file from Firebase storage, which I can do with other uploads — which makes me think it might be corrupted, or that there was some sort of problem during upload. I've gone back to Tony and asked if he can pass along the original file.

**Comment by sarahrainbow** [comment]
sorry just forwarded you Tony's feedback email with dropbox links to media he used!

**Comment by allishultes** [comment]
This looks like an issue with Quicktime / mov formats. See [this project](https://digital-paper-edit.test.tools.bbc.co.uk/#/projects/bMlNEs7e9rQ6f87nskhM) when signed into the admin account. 

When testing on quicktime files I've uploaded, I'm able to download them and play them back from the media export option, but not using the src url that the video player uses... 

https://developer.mozilla.org/en-US/docs/Web/Media/Formats/Containers

**Comment by bevand10** [comment]
The clue to the problem is in the filename - “ ProRes”.

I presume your backend tools can convert the content from Tony’s file to h264 for display in the browser?

--Dave


Dave Bevan
Senior Systems Engineer, News Labs & Tech Lead, Internet Fit Radio

BBC News Labs

+44 (0) 7703 54 66 92
New Broadcasting House | 2nd floor | Zone B | London W1A 1AA

@BBC_News_Labs<https://twitter.com/bbc_news_labs?lang=en>
bbcnewslabs.co.uk<https://bbcnewslabs.co.uk/>

On 12 Jul 2021, at 13:53, Sarah Rainbow ***@***.***> wrote:

﻿

low res footage uploaded by Tony Ellam 'TURBINES_FLEURY-ProRes Proxy half size.mov'
will only playback audio in both transcript editor and programme script view.

It seems he got low res version by exporting from Jupiter.

**Comment by sarahrainbow** [comment]
Yeah my first thought would be to add a video conversion step with ffmpeg

**Comment by allishultes** [comment]
> Yeah my first thought would be to add a video conversion step with ffmpeg

Yeah this is a good shout. @sarahrainbow and I had a lil chat and one possible complication around converting file formats is that the EDL and FCPX files need to reference whatever the original file on the user's machine is — which might be Quicktime (unless the user exports the media too). 

So potentially, we're talking converting the video for playback in the browser, but also storing a reference to the original file format so that the export EDL / FCPX files work. I don't _know_ if that's an issue, or if it would introduce other issues. If we're having to re-link files anyways after importing FCPX / EDL into editing software, does it matter what file extension the original reference had, or that the two aren't aligned? 

Re: the trial: the audio still plays back on these files and the EDL / FCPX / media export options still work. Obviously not ideal, but for now, if this effects one user, we could ask them to convert their file [using something like this](https://cloudconvert.com/mov-to-mp4) and re-upload it, and/or work without video...

**Comment by bevand10** [comment]
I guess a broader point here is that if TE exported the clip from Jupiter, then that clip will already have browse proxy available in Jupiter, which could be linked to, instead of localising and isolating things within DPE. Same goes for a Jupiter transcript actually - available in UCED via existing Jupiter->Transcript pipelines in either Transcriptor or Volt.
Perhaps one to discuss at some point.

**Comment by allishultes** [comment]
> I guess a broader point here is that if TE exported the clip from Jupiter, then that clip will already have browse proxy available in Jupiter, which could be linked to, instead of localising and isolating things within DPE. Same goes for a Jupiter transcript actually - available in UCED via existing Jupiter->Transcript pipelines in either Transcriptor or Volt.
> Perhaps one to discuss at some point.

Yeah, potentially one for a later date but definitely out of scope while the user trial is ongoing. Thanks Dave!
